Ludicrous-Ducks/Bangazon_django
# Correcting add to order, allowing multiple adds of the same item

# Description
This fixes a problem with the add to order view, wherein the order could only accept one of any particular product.

# Related Ticket(s)
#2 
#65 

## Testing
[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[X] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..

## Deploy Notes
The order model has changed, so a migration is necessary. A database rebuild may also be necessary.

## Steps to Test
The site is by now integrated. Start `python manage.py runserver` and navigate to a product detail page. Add it. Navigate back. Add it again. The resulting Order Detail redirect should contain two copies of the same product, as intended.
